content/blog/announcing-kustomize-support/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "Announcing Kustomize Support"
+date: 2020-07-06
+draft: true
+meta_desc: "Pulumi adds Kustomize support to their Kubernetes provider"
+meta_image: meta.png
+authors:
+    - levi-blackstone
+
+tags:
+    - kubernetes
+---
+
+Kubernetes is complex, and there are many ways to manage Kubernetes resources. Pulumi supports many of these options, including native code SDKs, YAML, Helm, and now, Kustomize.
```

Comment:
I'd like to see if we can make this more exciting for the reader because it's before the break and this is a super exciting feature. I'm happy to make a suggestion if needed. There's nothing wrong with this at all, just a suggestion.

Reply:
Sure, happy to take suggestions.

Reply:
Kubernetes is complex, and there are many ways to manage Kubernetes resources. Pulumi supports many of these options, including native code SDKs, YAML, Helm, and now, Kustomize. If you have existing kustomization.yaml files or use Kustomize to configure applications, you can reuse them wtih Pulumi’s infrastructure as code engine. You can efficiently adopt existing resources to deploy your modern application and save time and effort.